k9mail/src/test/java/com/fsck/k9/mailstore/MessageViewInfoExtractorTest.java

```diff
@@ -58,6 +59,8 @@
     public static final String BODY_TEXT = "K-9 Mail rocks :>";
     public static final String BODY_TEXT_HTML = "K-9 Mail rocks :&gt;";
     public static final String BODY_TEXT_FLOWED = "K-9 Mail rocks :> \r\nflowed line\r\nnot flowed line";
+    public static final String SUBJECT = "sabject";
```

Comment:
Typo?

Reply:
More of a habit, making symbols distinguishable from their content. Doesn't really matter in java I guess where prettyprinting objects isn't really a thing.